### PR TITLE
[chore] Fixed the "Getting In touch" URL

### LINKED ISSUE #10467: "Getting In Touch" references disused resource
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6574d2ffcbf2feb595e15eb3542c60e060136e9a/README.md?plain=1#L9

The "Getting In Touch" link in the README points to a Gitter.im channel that is clearly abandoned. That abandoned channel says to use an `#opentelemetry-collector` channel in a CNCF Slack instance. As far as I can tell, the channel in the Slack instance is actually `#otel-collector`.

The link in the README should point to a document that describes how to join the Slack instance and which channel to join upon connecting.

### PR DESCRIPTION
**Pull request by MalaSundaramoorthy**
**Description:** <Describe what has changed.>

Fixes #10467
**Motivation:**  Update the "Getting In Touch" link which was pointing to abandoned link.

**Link to tracking Issue:** <Issue number if applicable>  Issue : Getting In Touch" references disused resource #10467

**How it has been tested ** <Describe what testing was performed and which tests were added.>

- [x] Identified Below URL is the working one for opentelemetry-collector-contrib for  github main page 

https://github.com/open-telemetry/opentelemetry-collector-contrib

- [x] Tested on browser

### CODE REVIEW

**Comment by MalaSundaramoorthy** [comment]
@codeboten
This is my first time contribution. Help me to pass the above checks. Since my changes contributes to "README" file. Do I need to pass these checks?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
     <a href="https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md">Getting Involved</a>
     &nbsp;&nbsp;&bull;&nbsp;&nbsp;
-    <a href="https://gitter.im/open-telemetry/opentelemetry-service">Getting In Touch</a>
+    <a href="https://github.com/open-telemetry/opentelemetry-collector-contrib">Getting In Touch</a>
```
Comment (jpkrohling) [inline README.md:9]: This should probably be the slack channel, which replaced the Gitter one.

https://cloud-native.slack.com/archives/C01N6P7KR6W

Comment (MalaSundaramoorthy) [reply README.md]: Replaced the URL with the Slack channel url.

**Review by jpkrohling — APPROVED** [review]

**Comment by MalaSundaramoorthy** [comment]
@jpkrohling Would it possible to skip changelog entry on this PR?

**Comment by MalaSundaramoorthy** [comment]
@jpkrohling created a entry changelog in unreleased folder

File: unreleased/fix-broken-link.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```
Comment (jpkrohling) [inline unreleased/fix-broken-link.yaml:1]: Can you remove this file? I'll add a label to skip the changelog validation. I don't think this change should be part of the changelog.

Comment (MalaSundaramoorthy) [reply unreleased/fix-broken-link.yaml]: @jpkrohling  file removed.

**Comment by MalaSundaramoorthy** [comment]
@jpkrohling please review and help me to pass the checks. Thanks.

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
I'll merge this on green.

**Comment by MalaSundaramoorthy** [comment]
@jpkrohling Thank you for your help!

**Pull request merged**